workspaces/tarball/src/utils/analyzeDependencies.ts

```diff
@@ -72,27 +125,38 @@ export function analyzeDependencies(
   };
 }
 
-/**
- * @param {!string} moduleName
- * @returns {boolean}
- */
-function isNodeCoreModule(moduleName: string): boolean {
+function difference<T>(arr1: T[], arr2: T[]): T[] {
+  return arr1.filter(item => !arr2.includes(item));
+}
+
+function isFile(
+  name: string
+) {
+  return name.startsWith(".") || path.extname(name) !== ""
+}
+
+function isCoreModule(
+  moduleName: string
+): boolean {
   const cleanModuleName = moduleName.startsWith("node:") ? moduleName.slice(5) : moduleName;
 
   // Note: We need to also check moduleName because builtins package only return true for 'node:test'.
```

Comment:
We can remove/rewrite this comment as we don't use builtins package anymore

Reply:
No that's still valid, builtins here is not about the package name 😉

Reply:
oh, ok ^^